src/ex3/commands.js

```diff
@@ -0,0 +1,104 @@
+import { writeToFile, readFromFile } from "./fs.js";
+import {
+  checkItem,
+  capitalize,
+  convertToArray,
+  deletebyIndex,
+  deleteByName,
+} from "./utils.js";
+import { itemManager } from "../ex2/ItemManager.js";
+import { success, error, blue } from "./chalk.js";
+
+export async function addItem(item) {
+  let pokemonArray = [];
+
+  const inputArray = item.split(",");
+  inputArray.forEach(async (elm) => {
+    if (isNaN(elm)) {
```

Comment:
would be better to do the design in a way that you fetch the pokemon if it is a numebr and then just add it as a normal todo
you can think about this in a more general sense - there might be other weird types of todos i the future, atm it's just pokemon, but at the end of the day this just results in a text with a pokemon name, so would be more expandable to fetch this then add it normally